docs/eventing/sources/sinkbinding/README.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/
+  - /docs/eventing/sources/sinkbinding
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` object supports decoupling event production from
+delivery addressing.
+
+You can use sink binding to direct a subject to an event sink.
+A _subject_ is a Kubernetes resource that embeds a `PodSpec` template and produces events.
+An _event sink_ is an addressable Kubernetes object that can receive events.
+
+The SinkBinding object injects environment variables into the `PodTemplateSpec` of the
```

Comment:
formatting for object... I won't point out every instance but these are inconsistent throughout